thrift/src/main/thrift/Pinpoint.thrift

```diff
@@ -46,6 +46,12 @@ struct TJvmGc {
     5: i64          jvmMemoryNonHeapMax
     6: i64          jvmGcOldCount
     7: i64          jvmGcOldTime
+    8: i64          jvmPoolCodeCacheUsage
+    9: i64          jvmPoolEdenSpaceUsage
+    10: i64          jvmPoolOldGenUsage
+    11: i64          jvmPoolSurvivorSpaceUsage
```

Comment:
Marking these fields optional would be best in case something may change in the future.
```
jvmPoolCodeCacheUsage
jvmPoolEdenSpaceUsage
jvmPoolOldGenUsage
jvmPoolSurvivorSpaceUsage
```
The fields that are already there (`jvmGcOldCount`, `jvmGcOldTime`, etc) should have been optional as well imo, but it's too late to change that. We're trying to keep all new fields optional if possible.